### PR TITLE
feature(admin): added dashboard homepage template

### PR DESCRIPTION
**Pull request by WizBhoo**
Hi Yann,

Just added admin dashboard homepage.

Thanks

### CODE REVIEW

File: config/routes.yaml [hunk 1]
```diff
@@ -17,3 +17,8 @@ app_contact:
   path: /contact
   controller: App\Controller\ContactController::contact
   methods: GET|POST
+
+app_admin_dashboard:
+  path: /apiadmin/dashboard
```
Comment (drixs6o9) [inline config/routes.yaml:22]:
```suggestion
  path: /api-admin/dashboard
```
?

File: public/libs/dashboard/slick/config.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+css_dir = "."
+sass_dir = "."
+images_dir = "."
+fonts_dir = "fonts"
+relative_assets = true
+
+output_style = :compact
+line_comments = false
+
+preferred_syntax = :scss
```
Comment (drixs6o9) [inline public/libs/dashboard/slick/config.rb:10]:
```suggestion
preferred_syntax = :scss

```

File: templates/adminLayout.html.twig [hunk 3]
```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```
Comment (drixs6o9) [inline templates/adminLayout.html.twig:1]: Move it to templates/admin/layout.html.twig

File: templates/adminLayout.html.twig [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+        integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6"
+        crossorigin="anonymous">
+</script>
+<!-- Font-Awesome cdn js -->
+<script src="https://kit.fontawesome.com/58f30e32b1.js" crossorigin="anonymous"></script>
+<!-- libs JS -->
+<script src="{{ asset('libs/dashboard/slick/slick.min.js') }}"></script>
+<script src="{{ asset('libs/dashboard/wow/wow.min.js') }}"></script>
+<script src="{{ asset('libs/dashboard/animsition/animsition.min.js') }}"></script>
+<script src="{{ asset('libs/dashboard/wow/wow.min.js') }}"></script>
```
Comment (drixs6o9) [inline templates/adminLayout.html.twig:106]:
```suggestion
```

Already included.

**Review by drixs6o9 — CHANGES_REQUESTED** [review]

File: templates/adminLayout.html.twig (hunk 4 above)
Comment (WizBhoo) [reply templates/adminLayout.html.twig:106]: oups 👍

**Review by drixs6o9 — APPROVED** [review]

**Pull request merged**